src/main/java/xyz/vegaone/easytrackingv3/dto/Project.java

```diff
@@ -0,0 +1,28 @@
+package xyz.vegaone.easytrackingv3.dto;
+
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import xyz.vegaone.easytrackingv3.domain.SprintEntity;
+import xyz.vegaone.easytrackingv3.domain.TaskEntity;
+import xyz.vegaone.easytrackingv3.domain.UserEntity;
+
+import java.util.List;
+
+@Getter
+@Setter
+@ToString
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class, property = "id")
+public class Project {
+
+    private Long id;
+    private String projectName;
+    private String projectDescription;
```

Comment:
Could you please update the fields by removing project? So it will be just `name` and `description`.

Reply:
done